apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/IterableLikeToContainCheckerSamples.kt

```diff
@@ -5,6 +5,26 @@ import ch.tutteli.atrium.api.verbs.expect
 import kotlin.test.Test
 
 class IterableLikeToContainCheckerSamples {
+    @Test
+    fun exactly() {
+        expect(listOf("A", "B", "A")).toContain.inAnyOrder.exactly(2).entry {
+            toBeEqualComparingTo("A")
```

Comment:
Please use `toEqual` instead of toBeEqualComparingTo, there is not need for a compareTo comparison in case of a String:
```suggestion
            toEqual("A")
```
Same same further below

Reply:
Sure!